apps/evm/src/clients/api/mutations/updatePoolDelegateStatus/index.ts

```diff
@@ -0,0 +1,16 @@
+import { IsolatedPoolComptroller } from 'libs/contracts';
+
+export interface UpdatePoolDelegateStatusInput {
+  poolComptrollerContract: IsolatedPoolComptroller;
+  delegateAddress: string;
```

Comment:
I'd recommend using the term "delegatee" here (and in other places where we use the same word):
```suggestion
  delegateeAddress: string;
```